### DISCUSSION TITLE
Animation on input errors

### DISCUSSION THREAD

**Discussion by branislav-brincko** [mantinedev/mantine#6359] [Q&A]
Hello,

could someone recommend the cleanest way how to implement animations on input errors, for example fade in/slide in etc.?
I want to avoid a little jump of content below the input when the error is shown. 

Something like this works quite well for avoiding jump (and adding animation on opacity), but only on default sized input - to make it general, for all sizes and for all input types (number input, select, etc.) it get's complicated very quickly. 

![image](https://github.com/mantinedev/mantine/assets/118365024/3b7da0fd-322e-464e-aa2d-50ab8d5cda5d)

Any help appreciated 🙏

**Comment by yverby** [comment]
You can try the following
https://mantine.dev/core/input/#default-props-on-theme
```css
.root { ... }
.error { ... }
```
```jsx
import classes from './name.module.css';

export const theme = createTheme({
  components: {
    InputWrapper: Input.Wrapper.extend({
      classNames: {
        root: classes.root,
        error: classes.error,
      },
    }),
  },
});
```
